utils/docker-utils.sh

```diff
@@ -458,3 +460,55 @@ docker::scale_services_up() {
         overwrite "Waiting for $service_name to scale up ... Done"
     done
 }
+
+# Checks if the external networks exist and tries create it if not
+#
+# Arguments:
+# - $1 : path to the docker compose file with the network definitions
+#
+docker::ensure_external_networks_existence() {
+    if [[ -z "$1" ]]; then
+        log error "$(missing_param "ensure_external_networks_existence")"
```

Comment:
lets specify the param that is missing here.

Reply:
The current convention was that when it was just an array passed, the parameter name was not specified when log the error since there is no real name to bind the error to. However I do agree that it should specify some kind of name/indication as to what is missing.

So I updated all these cases to instead now be "[indicative name]", so just a name that makes sense for the list being passed in surrounded by [ ] to specify that it expects an array.